images/steam/Dockerfile

```diff
@@ -76,6 +81,10 @@ RUN rm /usr/bin/zenity && ln -s /usr/bin/true /usr/bin/zenity
 # refresh system font cache. For font warnings on startup see: https://github.com/ValveSoftware/steam-runtime/issues/482
 RUN fc-cache -f -v
 
+# set root password
+RUN echo "root:retro" | chpasswd
```

Comment:
I wouldn't do this, you can already open up a root shell using Docker
```
docker exec -u root -it <container_name> /bin/bash
```

Reply:
Sorry, I forgot to mention this change. Initially, this pull request was mostly intended for my personal use. Since when I connect in full screen via Moonlight, I prefer to operate within the terminal inside Moonlight. Therefore,  to make things easier for myself, I set a root password so I could switch to the root user.

If, in the future, there's support for keeping the container after closing an app, then the need to install applications inside the container will become more common. In that case, perhaps granting sudo privileges to the retro user would be more appropriate? (I noticed that the retro user is created in the 10-setup_user.sh script of the base image, without sudo privileges.)